kairon/shared/channels/whatsapp/bsp/dialog360.py

```
@@ -162,6 +162,9 @@ def list_templates(self, **kwargs):
         except DoesNotExist as e:
             logger.exception(e)
             raise AppException("Channel not found!")
+        except Exception as e:
+            logger.exception(e)
+            raise AppException(str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)